[UT] Run MaterializedViewFunctionTest failed

Run MaterializedViewFunctionTest can easily see such failed message, because the table doesn't complete add rollup operation.
``` 
[ERROR] testAggQuqeryOnAggMV5(org.apache.doris.planner.MaterializedViewFunctionTest)  Time elapsed: 0.57 s  <<< ERROR!
org.apache.doris.common.DdlException: errCode = 2, detailMessage = The table [emps]'s state is ROLLUP, cannot be dropped. please cancel the operation on olap table firstly. If you want to forcibly drop(cannot be recovered), please use "DROP table FORCE".
	at org.apache.doris.planner.MaterializedViewFunctionTest.afterMethod(MaterializedViewFunctionTest.java:85)
```

--- a/fe/fe-core/src/test/java/org/apache/doris/planner/MaterializedViewFunctionTest.java
+++ b/fe/fe-core/src/test/java/org/apache/doris/planner/MaterializedViewFunctionTest.java
@@ -82,9 +82,9 @@ public class MaterializedViewFunctionTest {
 
     @After
     public void afterMethod() throws Exception {
-        dorisAssert.dropTable(EMPS_TABLE_NAME);
-        dorisAssert.dropTable(DEPTS_TABLE_NAME);
-        dorisAssert.dropTable(USER_TAG_TABLE_NAME);
+        dorisAssert.dropTable(EMPS_TABLE_NAME, true);
+        dorisAssert.dropTable(DEPTS_TABLE_NAME, true);
+        dorisAssert.dropTable(USER_TAG_TABLE_NAME, true);
     }
 
     @AfterClass
@@ -607,7 +607,7 @@ public class MaterializedViewFunctionTest {
                 + "k1,k2;";
         String query = "select k1 from " + TEST_TABLE_NAME + " group by k1 having max(v1) > 10;";
         dorisAssert.withMaterializedView(createK1K2MV).query(query).explainWithout("k1_k2");
-        dorisAssert.dropTable(TEST_TABLE_NAME);
+        dorisAssert.dropTable(TEST_TABLE_NAME, true);
     }
 
     @Test
@@ -621,7 +621,7 @@ public class MaterializedViewFunctionTest {
                 + "k1,k2;";
         String query = "select k1 from " + TEST_TABLE_NAME + " group by k1 order by max(v1);";
         dorisAssert.withMaterializedView(createK1K2MV).query(query).explainWithout("k1_k2");
-        dorisAssert.dropTable(TEST_TABLE_NAME);
+        dorisAssert.dropTable(TEST_TABLE_NAME, true);
     }
 
     @Test
@@ -635,7 +635,7 @@ public class MaterializedViewFunctionTest {
                 + "k1,k2;";
         String query = "select k1 , sum(k2) over (partition by v1 ) from " + TEST_TABLE_NAME + ";";
         dorisAssert.withMaterializedView(createK1K2MV).query(query).explainWithout("k1_k2");
-        dorisAssert.dropTable(TEST_TABLE_NAME);
+        dorisAssert.dropTable(TEST_TABLE_NAME, true);
     }
 
     @Test
@@ -647,9 +647,7 @@ public class MaterializedViewFunctionTest {
                 + "k1;";
         String query = "select * from " + TEST_TABLE_NAME + ";";
         dorisAssert.withMaterializedView(createK1K2MV).query(query).explainContains(TEST_TABLE_NAME);
-        dorisAssert.dropTable(TEST_TABLE_NAME);
-
-
+        dorisAssert.dropTable(TEST_TABLE_NAME, true);
     }
 
     @Test
@@ -717,7 +715,7 @@ public class MaterializedViewFunctionTest {
         dorisAssert.withTable(aggTable);
         String query = "select k1, count(distinct v1) from " + TEST_TABLE_NAME + " group by k1;";
         dorisAssert.query(query).explainContains(TEST_TABLE_NAME, "bitmap_union_count");
-        dorisAssert.dropTable(TEST_TABLE_NAME);
+        dorisAssert.dropTable(TEST_TABLE_NAME, true);
     }
 
     @Test
@@ -727,7 +725,7 @@ public class MaterializedViewFunctionTest {
         dorisAssert.withTable(aggTable);
         String query = "select k1, count(distinct v1) from " + TEST_TABLE_NAME + " group by k1;";
         dorisAssert.query(query).explainContains(TEST_TABLE_NAME, "hll_union_agg");
-        dorisAssert.dropTable(TEST_TABLE_NAME);
+        dorisAssert.dropTable(TEST_TABLE_NAME, true);
     }
 
     @Test
@@ -831,6 +829,6 @@ public class MaterializedViewFunctionTest {
         dorisAssert.withMaterializedView(createMVSQL);
         String query = "select count(distinct salary) from agg_table;";
         dorisAssert.query(query).explainContains("mv");
-        dorisAssert.dropTable("agg_table");
+        dorisAssert.dropTable("agg_table", true);
     }
 }

--- a/fe/fe-core/src/test/java/org/apache/doris/utframe/DorisAssert.java
+++ b/fe/fe-core/src/test/java/org/apache/doris/utframe/DorisAssert.java
@@ -88,8 +88,12 @@ public class DorisAssert {
     }
 
     public DorisAssert dropTable(String tableName) throws Exception {
+        return dropTable(tableName, false);
+    }
+
+    public DorisAssert dropTable(String tableName, boolean isForce) throws Exception {
         DropTableStmt dropTableStmt =
-                (DropTableStmt) UtFrameUtils.parseAndAnalyzeStmt("drop table " + tableName + ";", ctx);
+                (DropTableStmt) UtFrameUtils.parseAndAnalyzeStmt("drop table " + tableName + (isForce ? " force" : "") + ";", ctx);
         Catalog.getCurrentCatalog().dropTable(dropTableStmt);
         return this;
     }

